performance/plugins.php

```diff
@@ -3,3 +3,20 @@
 // Plugin-specific tweaks go in here
 
 namespace Automattic\VIP\Performance;
+
+/**
+ * By default, the Co-Authors Plus plugin will query in a compatibility mode where it uses an expensive JOIN for post_author in it.
+ * This filter disables it and uses a simpler, taxonomy-only query.
+ * Run `wp co-authors-plus create-terms-for-posts` to ensure all authors have an author taxonomy attached to them.
+ * 
+ * @see https://lobby.vip.wordpress.com/2017/11/07/co-authors-plus-global-filter/
+ * 
+ * @return void
+ */
+function vip_coauthors_plus_should_query_post_author() {
+	if ( class_exists( 'CoAuthors_Plus' ) ) {
+		add_filter( 'coauthors_plus_should_query_post_author', '__return_false' );
```

Comment:
Would this filter ever run if the `CoAuthors_Plus` class _didn't_ exist?

Reply:
No, why should it?

Reply:
Ah I think he means it could drop the `class_exists( 'CoAuthors_Plus' )` check, and the whole wrapper even I suppose?

Guess the overhead for non-CAP sites would be the same, either `vip_coauthors_plus_should_query_post_author` is the extra filter being registered or `coauthors_plus_should_query_post_author` is.

Reply:
Ah, got it. I'm not great at reading between the lines 🙂